packages/apps/composer-app/src/App.tsx

```diff
@@ -3,82 +3,34 @@
 //
 
 import { ErrorBoundary } from '@sentry/react';
-import React, { useCallback } from 'react';
-import { HashRouter, useRoutes } from 'react-router-dom';
+import React from 'react';
+import { HashRouter } from 'react-router-dom';
 import { useRegisterSW } from 'virtual:pwa-register/react';
 
-import { fromHost, fromIFrame, Space } from '@dxos/client';
+import { fromHost, fromIFrame } from '@dxos/client';
 import { Config, Defaults, Dynamics } from '@dxos/config';
 import { log } from '@dxos/log';
 import {
-  AppLayout,
   ErrorProvider,
   Fallback,
   FatalError,
   GenericFallback,
-  ManageSpacePage,
-  RequireIdentity,
   ServiceWorkerToast,
-  SpacesPage,
-  useTelemetry,
-  translations
+  translations,
+  StatusIndicator2
```

Comment:
should we deal with this in a separate pr then?

Reply:
Yes, the next PR on my mind is the component reshuffle, where we can dedupe these and also move them into the correct package.